pkg/webui/console/views/gateway-general-settings/basic-settings-form/index.js

```diff
@@ -131,6 +138,12 @@ const BasicSettingsForm = React.memo(props => {
         component={Checkbox}
         label={sharedMessages.enabled}
         description={sharedMessages.requireAuthenticatedConnectionDescription}
+        />
+        <Form.Field
+        title={sharedMessages.lbsLNSSecret}
+        description={sharedMessages.lbsLNSSecretDescription}
+        name="lbs_lns_secret.value"
```

Comment:
Can we validate values for this field on the client?

Reply:
Yeah good one. So the API allows `2048` bytes. But since we convert to `Base64` we need an additional 33% (at max) capacity. So 66% of 2048 = 1351,68 so I've set the max length at `1350`.

Reply:
👍 Please add this as description for L42 in `basic-settings-form/validation-schema.js `